packages/main/src/themes/TabContainer.css

```diff
@@ -30,6 +30,14 @@
 	border-top: var(--_ui5_tc_header_border_bottom);
 }
 
+:host([header-background-design="Solid"]) .ui5-tc__header {
+	background-color: var(--sapObjectHeader_Background);
+}
```

Comment:
Since "Solid" is the default value, and `background-color: var(--sapObjectHeader_Background);` is already being set in `.ui5-tc__header`, this code is redundant.